docs/pages/build-reference/infrastructure.md

```diff
@@ -147,21 +147,28 @@ When selecting an image for the build you can use the full name provided below o
   - Hardware: Intel(R) Core(TM) i7-8700B CPU (6 cores/12 threads), 64 GB RAM
   - Build resource limits: 3 cores, 12 GB RAM
 - npm cache. [Learn more](caching/#javascript-dependencies)
+- CocoaPods cache. [Learn more](caching/#ios-dependencies)
 - `~/.npmrc`
 
   ```ini
-  registry=http://10.254.24.8:4873
+  registry=http://10.254.24.9:4873
   ```
 
 - `~/.yarnrc.yml`
 
   ```yml
   unsafeHttpWhitelist:
     - '*'
-  npmRegistryServer: 'registry=http://10.254.24.8:4873'
+  npmRegistryServer: 'registry=http://10.254.24.9:4873'
   enableImmutableInstalls: false
   ```
 
+- Project's `Podfile` will have CocoaPods cache server URL added as a source during the build step called `Install pods` (if `Podfile` already contains explicitly specified `https://cdn.cocoapods.org/` source, it will be replaced with CocoaPods cache server URL).
```

Comment:
```suggestion
- For iOS builds, EAS Build updates your `Podfile` during the `Install pods` build phase. We set the CocoaPods cache server URL as the `source`. If your `Podfile` includes `source "https://cdn.cocoapods.org/"`, it will be replaced with the CocoaPods cache server URL.
```